site/docs/patterns/international-address-input.mdx

```diff
@@ -0,0 +1,147 @@
+    contentPosition="top"
+    border
+  >
+    #### Spain
+  </Diagram>
+  <Diagram
+    src="/img/patterns/international-address-input/dynamic_united_kingdom.png"
+    alt="Example showing Dynamic template with United Kingdom selected"
+    contentPosition="top"
+    border
+  >
+    #### United Kingdom
+  </Diagram>
+</Diagrams>
+
+### Generic format
+
+Generic format provides a single template of international address input for all countries and regions.
+
+In some cases, it may be difficult to use templates due to requirements or product scope. In these cases, use Generic format.
+
+<Diagram
+  src="/img/patterns/international-address-input/generic_format.png"
+  alt="Example showing Generic format"
+  border
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+  - Choose the format that best fits the technical capabilities of your app,
+  delivery timescales, and required locations. - Combine Dynamic format and
+  Generic format if users will input international addresses across the world.
+  You can provide templates using both Generic and Dynamic formats. 
+</ExampleContainer>
+
+## Adding an additional address
+
+There may be cases where an additional address form is needed:
+
+- The client address may be different to the business address. We recommend a checkbox after the first form for the user to select if they need the additional form.
+- In some countries, such as China, users are required to input the address twice: once in English and then in Chinese. Add an additional form in the local language.
+
+<Diagram
+  src="/img/patterns/international-address-input/additional_address.png"
+  alt="Example showing additional address input"
+  border
+/>
+
+## Customized Address Forms
+
+There may be cases where you need to customize the address form considering other information you are collecting based on the product scope.
+
+- An additional form e.g. House No. may be needed.
+- The address form can be used alongside international phone number, email address, banking details, time zone selection, and other fields.
+
+<Diagram
+  src="/img/patterns/international-address-input/customized_address_form.png"
+  alt="Example showing a customized address form"
+  border
+/>
+
+## Toggling the local language
+
+You can add a switch to toggle localization after the country selection to enable users to compare the form with another address or data source.
+
+<Diagrams>
+  <Diagram
+    src="/img/patterns/international-address-input/toggle_language_1.png"
+    alt="Example showing a form with a show local language toggle off"
+    border
+  />
+  <Diagram
+    src="/img/patterns/international-address-input/toggle_language_2.png"
+    alt="Example showing a form with a show local language toggle on"
+    border
+  />
+</Diagrams>
+
+## Expanding the pattern
+
+If you find a need to expand the pattern, for example to have localized Dynamic templates for more countries or regions, please reach out to the team to discuss.
```

Comment:
need "reach out" link

Reply:
Link in Figma points to support page? https://www.saltdesignsystem.com/salt/support-and-contributions/index